lib/src/base-styles/_nx-page-header.scss

```
@@ -41,61 +41,60 @@ $main-header-product-grid-rows: $main-header-branding-height / 2;
   color: $nx-off-white;
   display: -ms-grid;
   display: grid;
-  grid-column-gap: 2px;
-  grid-row-gap: 2px;
-  -ms-grid-rows: $main-header-product-grid-rows 2px auto;
-  grid-template-rows: $main-header-product-grid-rows auto;
-  -ms-grid-columns: $main-header-branding-height 2px auto;
-  grid-template-columns: $main-header-branding-height auto;
+  grid-column-gap: $nx-spacing-xxs;
+  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
+  grid-template-columns: $main-header-product-height auto;
   grid-template-areas:
     "logo title"
     "logo version";
-  height: $main-header-branding-height;
+  height: $main-header-product-height;
   margin-top: 10px;
   padding-right: 12px;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
 .nx-product__logo {
+  -ms-grid-row: 1;
+  -ms-grid-row-span: 2;
+  -ms-grid-column: 1;
   -ms-grid-row-align: center;
   align-self: center;
   box-sizing: border-box;
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 3;
-  -ms-grid-column: 1;
   grid-area: logo;
-  height: $main-header-branding-height;
-  width: $main-header-branding-height;
+  height: $main-header-product-height;
+  width: $main-header-product-height;
 }
 
 .nx-product__name {
+  -ms-grid-row: 1;
+  -ms-grid-column: 3;
   @include semi-bold();
 
-  -ms-grid-row-align: end;
-  align-self: end;
+  -ms-grid-row-align: center;
+
+  align-self: center;
   font-size: 23px;
-  text-transform: lowercase;
-  -ms-grid-row: 1;
-  -ms-grid-column: 3;
   grid-area: title;
   -ms-grid-column-align: left;
   justify-self: left;
-  line-height: 18px;
+  line-height: 100%;
```

Not that it matters for anything except the boxes as seen in the dev tools, but this seems to work best:

```suggestion
  line-height: normal;
```